core/device_hooks/reference_hooks.cpp

```diff
@@ -30,7 +30,6 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-
 #include <ginkgo/core/base/version.hpp>
```

Comment:
I could not find this rule of "single empty line after license beacon" . Maybe you can add it to the contribution guidelines ?

Reply:
We actually don't have this rule and we have both versions (1 and 2 empty lines) present in our files. I think it is fine to allow and have both in our files (the contributing guidelines are quite loaded already).

Reply:
add_license.sh use single empty line after license.
I am fine with whether add the rule into guidelines or not, but I would like to keep the same behavior as the current script.

Reply:
I am fine with having just one empty line (and that the script does it that way), I would just like to not add it to the guidelines because it is quite unimportant in my opinion, and I don't want to increase the guidelines with such a detail.